main.go

```diff
@@ -380,6 +397,12 @@ func main() {
 
 	var r registry.Registry
 	switch cfg.Registry {
+	case "dynamodb":
```

Comment:
I think we should prefix with provider like aws-dynamodb.

Reply:
DynamoDB is a registered trademark of Amazon Technologies, Inc. The term is not ambiguous.

Reply:
Not finding much precedent either way. The only double-barrelled registry/provider I can find is awssd/aws_sd. But "Service Discovery" is a generic term and there's another AWS provider.

On the other side there's ns1, which has no "ibm" prefix. But that's probably historical, as there was an acquisition.

So I'm of two minds as to whether "DynamoDB" is sufficiently famous or needs qualification with "AWS". I'm not quite sure we'd want to call this simply "AWS".

Reply:
then it sounds also fine to me